persist/sqlite/consensus.go

```diff
@@ -644,15 +826,23 @@ func (s *Store) LastCommittedIndex() (index types.ChainIndex, err error) {
 	return
 }
 
+// ResetLastIndex resets the last indexed tip to trigger a full rescan.
+func (s *Store) ResetLastIndex() error {
```

Comment:
Consider moving this to a `_test.go` file if it's only supposed to be called in testing.